_config.yml

```diff
@@ -21,6 +21,12 @@ description: >-
 url: "https://eips.ethereum.org"
 twitter_username: ethereum
 github_username:  ethereum
+youtube_username: channel/UCNOfzGXD_C9YMYmnefmPH0g
```

Comment:
This is the EF's channel, not an EIPIP specific channel.  Same with GitHub username and Twitter username though... what is the purpose of all of these things being in here?

Reply:
I'd either add the youtube link as that includes ACD, Eth2 calls, etc. or remove github/twitter/youtube.

Reply:
@Souptacular @nicksavers any opinions?

Reply:
I think we should remove GitHub/YouTube/Twitter until we have a more neutral place that the videos belong to. Linking official EF with EIPs has bad optics at minimum. I do understand my previous sentence has the implication that the ACD calls should be moved off of the EF YouTube channel and I agree with that, but it hasn't happened yet. As far as the `ethereum` repo being primarily controlled by the EF and having a mix of EF and non-EF repos, we are in the process of rectifying that.

Reply:
Reopened this PR because it seems it was never followed up on.

1. Should all these social media links removed them?
2. With the exception of GitHub since the repo is there?